VorpalEngine/sources/utility/asset/Storage.cpp

```diff
@@ -109,27 +120,31 @@ Handler Storage::ProcessHandlerCreation(const KeyType& key)
 
         if (requestFlag.second)
         {
-            LOG_DEBUG("Processing sync task \"%s\" -> creating entry", key.c_str());
+            LOG_DEBUG(
+                "Processing sync task \"%s\" -> creating entry", key.c_str());
 
             m_entries.insert(std::make_pair(key, promise.get_future().share()));
         }
     }
 
     if (requestFlag.second)
     {
-        LOG_DEBUG("Processing sync task \"%s\" -> creating handler", key.c_str());
+        LOG_DEBUG(
+            "Processing sync task \"%s\" -> creating handler", key.c_str());
 
         Handler handler = CreateHandler(key);
 
-        LOG_DEBUG("Processing sync task \"%s\" -> publishing handler to entry", key.c_str());
+        LOG_DEBUG("Processing sync task \"%s\" -> publishing handler to entry",
+            key.c_str());
 
         promise.set_value(handler);
 
         return handler;
     }
     else
     {
-        //! If given key was already requested, we need to either snatch the order
+        //! If given key was already requested, we need to either snatch the
+        //! order
```

Comment:
welp that was silly. Should've combined both comments before breaking the first one